test/new/redos-html-closing.md

```diff
@@ -0,0 +1 @@
+<tag		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""		""<"	/>a'a
```

Comment:
can we use spaces instead of tabs?
why is redos-html-closing.html empty?
can you use underscores instead of dashes in the test name to conform to other tests?

Reply:
1. Can we create an issue to examine that question? (Not sure if they absolutely need to be tabs or not, would like to know the answer.)
2. See other comment.
3. Recommend fixing that in a different PR? (@davisjam contributed a lot today and it seems like relatively small things compared to the bigger problem. He called me out on the same thing when I was trying to make the testing harness prettier - and even did follow up on that.)

These don't seem like dealbreakers for merging to me right now.

Reply:
ps. @UziTech and I made the testing harness prettier at the same time and those changes have already been merged.

Reply:
> can we use spaces instead of tabs?

Why?

>  why is redos-html-closing.html empty?

See below; I don't know what the expected output is.

>  can you use underscores instead of dashes in the test name to conform to other tests?

Yes, will fix.